content/en/blog/falco-0-37-0/index.md

```diff
@@ -0,0 +1,296 @@
+---
+title: Introducing Falco 0.37.0
+date: 2024-01-30
+author: Luca Guerra, Andrea Terzolo, Mike Coleman
+slug: falco-0-37-0
+tags: ["Falco","Release"]
+---
+
+Dear Falco Community, today we are happy to announce the release of Falco 0.37.0!
+
+This release brings an improved installation experience, a new way to modify falco rules, and some great UX improvements. There are, as to be expected, a handful of breaking changes. But, rest assured, we've done all we can to help you with any changes you might need to make.
+
+
+
+`===============================`
+
+**Need to update these stats before publishing**
+
+*During this release cycle, we merged more than 100 PRs on Falco and more than 150 PRs for libs and drivers, version 0.13.1 and version 6.0.1 respectively. Thank you to our maintainers and contributors, as this would not happen without your support and dedication!*
```

Comment:
How do we get these, you may be asking?

In libs, you go to PRs and enter the search query: `is:pr is:closed milestone:0.14.0` (https://github.com/falcosecurity/libs/pulls?q=is%3Apr+is%3Aclosed+milestone%3A0.14.0+) and then 0.14.1 and 0.14.2 because those are the ones that will get into this release (you can double check by going to the libs repo in the latest released versions), then add the numbers and you'll get the total. Same for Falco with version 0.37.0

```suggestion
*During this release cycle, we merged more than 100 PRs on Falco and more than 160 PRs for libs and drivers, version 0.14.2 and version 7.0.0 respectively. Thank you to our maintainers and contributors, as this would not happen without your support and dedication!*
```